packages/runtime/src/helpers/functionsMetaData.ts

```diff
@@ -15,6 +15,19 @@ const getNextRuntimeVersion = async (packageJsonPath: string, useNodeModulesPath
   return useNodeModulesPath ? packagePlugin.version : packagePlugin.dependencies[NEXT_PLUGIN]
 }
 
+const pluginPackagePath = '.netlify/plugins/package.json'
```

Comment:
Use the constant format for constant variables to keep the codebase uniform.

```suggestion
const PLUGIN_PACKAGE_PATH = '.netlify/plugins/package.json'
```